source/contracts/trading/ClaimProceeds.sol

```diff
@@ -51,11 +56,19 @@ contract ClaimProceeds is CashAutoConverter, ReentrancyGuard, IClaimProceeds {
             if (_reporterShare > 0) {
                 require(_denominationToken.transferFrom(_market, _market.getUniverse().getNextReportingWindow(), _reporterShare));
             }
+            
+            _numShares = _numShares.add(_numberOfShares);
+            _numPayoutTokens = _numPayoutTokens.add(_shareHolderShare);
         }
 
+        logProceedsClaimed(_market, msg.sender, _numShares, _numPayoutTokens);
         return true;
     }
 
+    function logProceedsClaimed(IMarket _market, address _sender, uint256 _numShares, uint256 _numPayoutTokens) private returns (bool) {
```

Comment:
Is this a separate function due to stack depth issues?  If not, why not just inline the log call like everywhere else?

Reply:
Yuuup. Stack depth fun.